docs/src/components/footer/EmailSubscribe.tsx

```diff
@@ -64,7 +64,7 @@ export default function EmailSubscribe({ sx }: { sx?: SxProps<Theme> }) {
               sx={{
                 color: (theme: Theme) =>
                   theme.palette.mode === 'dark'
-                    ? theme.palette.success[500]
+                    ? theme.palette.success[600]
```

Comment:
@siriwatknp @oliviertassinari 
Using this small tweak to open a thread about the email subscription:

> Email Subscribe UX the code is copied from old rebranding. If you subscribe, you will see a green alert (replace the input) tell u to check your inbox. Do we want to go with this UX?

I personally don't have anything against this. I have already tweaked the alert styles and I think it's okay. Do you have anything else in mind?!

> Confirm page (once click subscribe confirmation in email)? I think we need this page, can you do a quick design on this?

Yeah, indeed! It'd be nice to also make something more custom for the subscription confirmation email:

<img width="500" alt="Screen Shot 2021-08-24 at 13 38 59" src="https://user-images.githubusercontent.com/67129314/130655909-20d5df5f-3ee3-4747-9287-d385e3b385c5.png">

Something very very simple but elegant will do the job. Once I have something designed, I'll paste the Figma link here.